webpack/assets/javascripts/react_app/routes/Models/consts.js

```diff
@@ -0,0 +1,19 @@
+
+export default {
+  controller: "models",
+  autocomplete : {
+    searchQuery: null,
+    url: "/models/auto_complete_search",
+    id: "searchBar",
+    useKeyShortcuts: true
+  },
+  bookmarks: {
+    url: "/api/bookmarks",
```

Comment:
Should this also use the URL helper?

Reply:
This gets passed only to search component, so I think it is ok as is.